src/Components/Actions/AddressAutocompleteAction/AddressAutocompleteAction.tsx

```diff
@@ -58,6 +70,12 @@ const FakeInput = styled(Input)`
   padding: 0 16px;
   line-height: 1;
   font-size: 48px;
+  color: ${colorsV3.gray900};
+
+  &:disabled {
+    -webkit-text-fill-color: ${colorsV3.gray900};
+    opacity: 1;
```

Comment:
This seems to fix the disabled placeholder color?
```suggestion
    opacity: 1;
    ::placeholder {
      -webkit-text-fill-color: ${colorsV3.gray500};
    }
```

Reply:
It's so weird, I tried that, but I can see that it updates the placeholder either. Does it work for you?

Reply:
This is the downside of using a disabled input, since we need to set the opacity and add -webkit-text-fill-color to style the placeholder and input text in safari. According to mozilla it's not optimal https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-fill-color

I think I can live with that but it might be better to use a standard div instead, WDYT?

Reply:
Yeah, most likely it's a better option long term. I thought it would be the easiest way to inherit all the styles instead of redefining them but it might turn out not to be the case!

Reply:
I'm fine either way, that 👆 works in Safari as well (both desktop and mobile) with Storybook at least